### PR TITLE
docs: Convert docs to typescript

### PR DESCRIPTION
**Pull request by ling1726**

### CODE REVIEW

File: apps/website/src/components/Playground/index.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { SandpackProvider, SandpackLayout, SandpackCodeEditor, SandpackPreview } from '@codesandbox/sandpack-react';
 import { useColorMode } from '@docusaurus/theme-common';
-import AppCode from '!!raw-loader!./template/App.js';
-import StylesCode from '!!raw-loader!./template/Styles.js';
+import AppCode from '!!raw-loader!./template/App.tsx';
```
Comment (ling1726) [inline apps/website/src/components/Playground/index.tsx:4]: it's possible to keep the playground in JS, it was just easier to turn them into TS files too... but I'm open to doing extra work to leave it as JS too

Comment (layershifter) [reply apps/website/src/components/Playground/index.tsx:4]: I briefly checked and it seems that sandpack does not emit TS errors. So may be there is sense to keep them in JS. At least it feels that `react-preset` loads faster :)

Comment (ling1726) [reply apps/website/src/components/Playground/index.tsx:4]: that's fine, we can still type check internally with jsdoc for these files

Comment (ling1726) [reply apps/website/src/components/Playground/index.tsx:4]: I guess if we ever get more complicated requirements for live code editing we'll have to investigate docusaurus live editing

File: apps/website/src/theme/Toggle/index.tsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ function GithubLink() {
 
 export default function () {
   return (
-    <div className={styles.container}>
+    <div className={styles['container']}>
```
Comment (ling1726) [inline apps/website/src/theme/Toggle/index.tsx:52]: @layershifter  is there a reason the tsconfig you originally proposed bans `styles.container` ?

Comment (layershifter) [reply apps/website/src/theme/Toggle/index.tsx:52]: Not really, AFAIR it's defaults from Nx setup

**Review by layershifter — APPROVED** [review]

**Pull request merged**